packages/kit/src/states/jotai/contexts/swap/actions.ts

```
@@ -176,13 +176,19 @@ class ContentJotaiActionsSwap extends ContextJotaiActionsBase {
-          if (defaultTokenSet.fromToken) {
-            set(swapSelectToTokenAtom(), defaultTokenSet.fromToken);
-          }
+          set(
+            swapSelectToTokenAtom(),
+            !defaultTokenSet.toToken?.isNative
+              ? defaultTokenSet.toToken
+              : undefined,
+          );
+        } else if (
+          defaultTokenSet.fromToken &&
+          defaultTokenSet.fromToken.isNative
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)